symbolic-unwind/src/evaluator/parsing.rs

```diff
@@ -198,39 +188,28 @@ fn base_expr<T: RegisterValue>(input: &str) -> IResult<&str, Expr<T>, ParseExprE
     ))(input)
 }
 
-/// Parses a stack of [expressions](super::Expr).
-///
-/// # Example
-/// ```rust
-/// use symbolic_unwind::evaluator::parsing::expr_stack;
-/// use symbolic_unwind::evaluator::{BinOp, Expr};
-///
-/// let (_, stack) = expr_stack::<u8>("1 2 + 3").unwrap();
-/// assert_eq!(stack.len(), 2);
-/// assert_eq!(
-///     stack[0],
-///     Expr::Op(
-///         Box::new(Expr::Value(1)),
-///         Box::new(Expr::Value(2)),
-///         BinOp::Add
-///     )
-/// );
-/// assert_eq!(stack[1], Expr::Value(3));
-/// ```
-pub fn expr_stack<T: RegisterValue>(
-    mut input: &str,
-) -> IResult<&str, Vec<Expr<T>>, ParseExprError> {
+/// Parses an [expression](super::Expr).
```

Comment:
You had a great example on `expr_stack`, would it be worth retaining it in a modified form for `expr`?

Reply:
Ah, I removed it because I thought it was obvious what `expr` does (as opposed to how it does it internally, which is … not great). I can add one if you like.